llama_stack/apis/tools/tools.py

```diff
@@ -87,7 +89,6 @@ class Tools(Protocol):
     @webmethod(route="/toolgroups/register", method="POST")
     async def register_tool_group(
         self,
-        name: str,
```

Comment:
this identifier must be here since that's what registration and unregistration will be done with